.fides/db_dataset.yml

```diff
@@ -2227,6 +2227,8 @@ dataset:
       data_categories: [system]
     - name: user_assigned_data_categories
       data_categories: [system]
+    - name: hidden
```

Comment:
May need to go add something like this in F+, mostly making a note to remind myself